asr/wenet/cli/reverb.py

```diff
@@ -0,0 +1,392 @@
+import copy
+from functools import cached_property, lru_cache, partial
+import logging
+import os
+from itertools import groupby, chain
+from pathlib import Path
+from math import ceil
+from typing import List
+import shutil
+import yaml
+
+import torch
+import torch.nn.functional as F
+
+import torchaudio
+from torchaudio.compliance import kaldi
+
+from wenet.cli.utils import hyps_to_ctm, hyps_to_txt
```

Comment:
This file doesn't exist in this commit

Reply:
Thanks for catching that! Just added the file that I missed!